lib/spack/spack/audit.py

```diff
@@ -289,9 +289,13 @@ def _check_build_test_callbacks(pkgs, error_cls):
         pkg_cls = spack.repo.path.get_pkg_class(pkg_name)
         test_callbacks = getattr(pkg_cls, "build_time_test_callbacks", None)
 
-        if test_callbacks and "test" in test_callbacks:
-            msg = '{0} package contains "test" method in ' "build_time_test_callbacks"
-            instr = 'Remove "test" from: [{0}]'.format(", ".join(test_callbacks))
+        # TODO: change "test" and "test*" to "test_" and "test_*" once remove "test"
```

Comment:
Is this a TODO for this PR, of for a next version of Spack (e.g. after a deprecation period)?

Reply:
For next iteration after a deprecation period.

Reply:
For these TODOs (including others in the PR), can we add something to make them greppable in groups? I was thinking something trivial like:
```suggestion
        # TODO (stand-alone test like pytest): change "test" and "test*" to "test_" and "test_*" once remove "test"
```
or any other unique identifier for this set of changes. The rationale is that, when we want to take care of them after v0.20, they can be spotted easily in the code.

Reply:
How about "TODO (post-34236)"?  I made that change and noticed another place where I forgot one.

Reply:
Sure, anything uniquely identifiable is fine with me.